### ISSUE TITLE
fix: the badge of the GitHub stars is render on the page of each component

### ISSUE THREAD

**Issue by TahimiLeonBravo** [nexxtway/react-rainbow#481]
- Current behavior:
When change between pages in the library (e.g when you are in components page and select a specific component) the GitHub Stars component (`library/styleguideComponents/SectionHeading/GithubStarsWrapper/index.js`) is recreated and thus it fetches the stars again.
- Expected behavior:
It should fetch GitHub stars only once (when the library first render)

**Comment by MarcosVillacanas** [comment]
Hi! Is this still causing trouble? If so, Could you please give me more context of the problem?

**Comment by LeandroTorresSicilia** [comment]
@MarcosVillacanas I just added a description of the bug above
Hint: since this library is based on react-styleguidist, their SectionHeading component is rerendered each time when you navigate, there is a RibbonRenderer component which is meant for this, then we must replace this component with ours.

**Comment by LeandroTorresSicilia** [comment]
We have this already done here
https://github.com/reiniergs/react-rainbow-styleguide

**Comment by MarcosVillacanas** [comment]
Okay, I'm gonna take it :)
I will let you know as soon as I have got something

**Comment by MarcosVillacanas** [comment]
I think I have solved the issue: https://github.com/nexxtway/react-rainbow/pull/1653
Could you please check it?
